__tests__/Snapshottest/__snapshots__/UIOutputComparisonTests.tsx.snap

```diff
@@ -146,6 +146,11 @@ exports[`LibraryContainer Test UI rendering of Library Item and child components
         onError={[Function]}
         src="test-file-stub"
       />
+      <img
```

Comment:
so I think you might need to update the tests file after you merge with master as I think this will undo what Keith added.

Reply:
ah..I think I discarded that change. Will check the master file.